src/providers/aws/specs/ec2fleet/default.json

```diff
@@ -43,25 +43,25 @@
     {
       "ResourceType": "fleet",
       "Tags": [
-        {"Key": "Name", "Value": "{{ fleet_name }}"},
-        {% for tag in base_tags %}
-        {"Key": "{{ tag.key }}", "Value": "{{ tag.value }}"}{% if not loop.last or has_custom_tags %},{% endif %}
-        {% endfor %}{% if has_custom_tags %},
-        {% for tag in custom_tags %}
-        {"Key": "{{ tag.key }}", "Value": "{{ tag.value }}"}{% if not loop.last %},{% endif %}
-        {% endfor %}{% endif %}
+        {"Key": "Name", "Value": "{{ fleet_name }}"}
```

Comment:
We might need to make other changes if we are doing it this way, ie we might not need `has_custom_tags` annymore?